src/Chirp.Infrastructure/Repositories/CheepRepository.cs

```diff
@@ -79,5 +79,16 @@ public async Task<IEnumerable<Cheep>> GetByAuthors(IEnumerable<Guid> authorIds,
 
             return cheeps;
         }
+
+        public async Task AnonymizeCheeps(Author author) {
```

Comment:
This creates a new Author per cheep from the deleted user. Should only create one new author.

Could be done by refactoring out the "new Author {}": 
```
var anonymousAuthor = new Author { Name = "[Deleted user]" };

foreach (Cheep loopCheep in cheeps) {
    loopCheep.Author = anonymousAuthor;
}
```